pkg/crypto/keys/publickey_test.go

```diff
@@ -57,7 +75,27 @@ func TestDecodeBytes(t *testing.T) {
 	decodedPubKey := &PublicKey{}
 	err := decodedPubKey.DecodeBytes(pubKey.Bytes())
 	require.NoError(t, err)
-	require.Equal(t, pubKey,decodedPubKey)
+	require.Equal(t, pubKey, decodedPubKey)
+}
+
+func TestSort(t *testing.T) {
+	unsorted := make(PublicKeys, 10)
+	for i := range unsorted {
+		priv, err := NewPrivateKey()
+		require.NoError(t, err)
+		unsorted[i] = priv.PublicKey()
+	}
+
+	pubs := make(PublicKeys, len(unsorted))
+	copy(pubs, unsorted)
+
+	sort.Sort(pubs[:])
+	require.NotEqual(t, unsorted, pubs)
```

Comment:
Weak check, you need to check for elements order.

Reply:
The point is that there is no particular order that we should rely upon in tests. The only thing we expect from sort function is to produce the same result on the same set of values.
I can shuffle array a bit several times though.